packages/firebase_crashlytics/ios/Classes/FirebaseCrashlyticsPlugin.m

```diff
@@ -0,0 +1,75 @@
+#import "FirebaseCrashlyticsPlugin.h"
+
+#import <Firebase/Firebase.h>
+
+@interface FirebaseCrashlyticsPlugin ()
+@property(nonatomic, retain) FlutterMethodChannel* channel;
+@end
+
+@implementation FirebaseCrashlyticsPlugin
++ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
+  FlutterMethodChannel* channel =
+      [FlutterMethodChannel methodChannelWithName:@"plugins.flutter.io/firebase_crashlytics"
+                                  binaryMessenger:[registrar messenger]];
+  FirebaseCrashlyticsPlugin* instance = [[FirebaseCrashlyticsPlugin alloc] init];
+  [registrar addMethodCallDelegate:instance channel:channel];
+}
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    if (![FIRApp defaultApp]) {
+      [FIRApp configure];
+    }
+  }
+  return self;
+}
+
+- (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
+  if ([@"Crashlytics#onError" isEqualToString:call.method]) {
+    NSError* error = [NSError errorWithDomain:call.arguments[@"exception"]
+                                         code:(int)call.arguments[@"code"]
+                                     userInfo:@{
+                                       @"exception" : call.arguments[@"exception"],
+                                       @"stackTrace" : call.arguments[@"stackTrace"]
+                                     }];
+    [[Crashlytics sharedInstance] recordError:error];
+    result(@"Error reported to Crashlytics.");
+  } else if ([@"Crashlytics#isDebuggable" isEqualToString:call.method]) {
```

Comment:
nit: I would call it isDebugMode instead of isDebuggable

Reply:
This is the name used by Android so I was sticking to that.

https://firebase.google.com/docs/reference/android/io/fabric/sdk/android/fabric/Fabric#isDebuggable()